lib/utils/formatTranslationChunks.js

```diff
@@ -0,0 +1,16 @@
+import { isArray } from 'lodash';
+
+/**
+ * Joins translation chunks from array into single string.
+ *
+ * @param {array} chunks
+ * @return {string} joined translation string
+ */
+
+export const formatTranslationChunks = (chunks = []) => {
```

Comment:
Please add tests for this method. See how it is done [here](https://github.com/folio-org/stripes-data-transfer-components/tree/master/lib/utils/tests).